.changelog/43356.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:new-resource
+aws_cognitoidp_log_delivery_configuration
+```
+
+```release-note:note
+resource/aws_cognitoidp_log_delivery_configuration: Because we cannot easily test all this functionality, it is best effort and we ask for community help in testing
```

Comment:
```suggestion
resource/aws_cognito_log_delivery_configuration: Because we cannot easily test all this functionality, it is best effort and we ask for community help in testing
```